vivisect/__init__.py

```diff
@@ -783,10 +783,12 @@ def followPointer(self, va):
             return True
 
         elif ltype == LOC_STRING:
+            logger.debug("followPointer->makeString(0x%x)", va)
```

Comment:
Should we just have one logger.debug call for all calls to followPointer so we can also catch erroneous calls that end up in NOPs??

Reply:
i'm not sure i understand.  i'm open to suggestions.

my primary goal is making sure we catch the linkage (what causes the creation of new objects) to make it easier to troubleshoot odd behaviors (like pointers that should be code, or things like that).  these are often nuanced buts in the disassembler, but irritating to track down, especially if we don't have the binaries (eg. other users reporting bugs).

Reply:
after re-reading your suggestion, i think i'm inclined to suggest instead that we have a "catch-all" log message.  i wouldn't want to remove these specific log messages, which provide more significant details for nearly-zero cost.